internal/tools/aws/tags_test.go

```diff
@@ -53,8 +57,9 @@ func TestTagsASRDSTags(t *testing.T) {
 	key2 := "key2"
 	value2 := "value2"
 	tags, err := NewTags(key, value, key2, value2)
+	result := tags.ToRDSTags()
 	assert.NoError(t, err)
```

Comment:
It is preferable to check error immediately after it occurs.

In this case, just after
```
tags, err := NewTags(key, value, key2, value2)
```